spec/rest/tls.test.js

```diff
@@ -0,0 +1,118 @@
+"use strict";
+
+define(['ably', 'shared_helper', 'async'], function(Ably, helper, async) {
+	var currentTime, exports = {};
+
+	/*
+	 * Set up and time-sync.
+	 */
+	exports.setupauth = function(test) {
+		test.expect(1);
+		helper.setupApp(function() {
+			var rest = helper.AblyRest();
+			(function(callback) {
+				rest.time(function(err, time) {
+					if(err) { callback(err); }
+					callback(null, time);
+				});
+			})(function(err, time) {
+				if(err) {
+					test.ok(false, helper.displayError(err));
+					test.done();
+					return;
+				}
+				currentTime = time;
+				test.ok(true, 'Obtained time');
+				test.done();
+			});
+		});
+	};
+
+	/*
+	 * tls:true initiates https connection
+	 */
+	exports.tls_connection = function(test) {
+		test.expect(4);
+
+		var rest,
+		keyStr = helper.getTestApp().keys[1].keyStr;
+
+		rest = helper.AblyRest({ tls: true, key: keyStr });
+
+		async.parallel([
+			function(cb) {
+				rest.auth.requestToken(function(err, tokenDetails) {
+					test.ok( tokenDetails.token, 'Verify that token is returned');
+					cb();
+				});
+			},
+			function(cb) {
+				rest.stats(function(err, stats) {
+					test.ok(err, 'Verify that an error was returned')
```

Comment:
It passed because the key `helper.getTestApp().keys[1].keyStr;` may not have had permissions to test for stats as it has less privileges, but that had nothing to do with the test you are executing unfortunately